### PR TITLE
PCT-1320 Fixed error in percona agent installer

### PR DESCRIPTION
**Pull request by cfsalguero**
The reported error was: "percona-agent doesn't use -agent-mysql-user and -agent-mysql-pass on non-interactive installation"

The colon at line 68 was creating a local "dsn" variable so the original one remained untouched without the correct user & apss provided in the command line.

### CODE REVIEW

**Comment by arvenil** [comment]
My friend, can we write a test for `-agent-mysql-user` and `-agent-mysql-pass`?

**Comment by arvenil** [comment]
:sparkles:  :sparkles:  :+1: :sparkles:  :sparkles: 
Awesome to merge :)

**Comment by daniel-nichter** [comment]
Nice catch. Literally a single-character bug. :-)

**Comment by arvenil** [comment]
@cfsalguero What happen to that PR?

File: bin/percona-agent-installer/main_test.go [hunk 1]
```diff
@@ -295,7 +297,7 @@ func (s *MainTestSuite) TestNonInteractiveInstallWithJustCredentialDetailsFlags(
 		"-basedir="+pct.Basedir.Path(),
 		"-api-host="+s.fakeApi.URL(),
 		"-interactive=false",
-		"-mysql-defaults-file="+test.RootDir+"/installer/my.cnf-wrong_user",
+		"-mysql-defaults-file="+test.RootDir+"/installer/my.cnf-root_user",
```
Comment (arvenil) [inline bin/percona-agent-installer/main_test.go]: Why this change?

**Comment by arvenil** [comment]
:sparkles: :sparkles: :+1: :sparkles: :sparkles:
